timetracker/entries/models.py

```diff
@@ -2,24 +2,39 @@
 
 from django.utils import timezone
 
+# One Project may have multiple clients, or two Projects may have one client, causing redundancy. To denormalise the table we isolate Project.client from the Projects model by creating a new model of Clients with which projects share a many-many relationship.
+
+class Client(models.Model):
+    name = models.CharField(max_length=200)
+
+    #how to create a method for displaying foreign key attributes? eg: a client's projects?
+
+    def __str__(self):
+        return '{}'.format(self.name)
 
 class Project(models.Model):
-    client = models.CharField(max_length=200)
     name = models.CharField(max_length=200)
+    client = models.ManyToManyField(Client, related_name="+")
```

Comment:
You shouldn't set `related_name="+"` here. You don't actually need a related_name at all in this instances.

Settings it to `+` stops Django from creating the reverse relationship between `Client` and `Project`, which is the problem you've highlighted with your comment in the `Client` model.